keps/0000-kep-template.md

```diff
@@ -135,13 +171,95 @@ What are the risks of this proposal and how do we mitigate.
 Think broadly.
 For example, consider both security and how this will impact the larger kubernetes ecosystem.
 
-## Graduation Criteria
+How will security be reviewed and by whom?
+How will UX be reviewed and by whom?
 
-How will we know that this has succeeded?
-Gathering user feedback is crucial for building high quality experiences and SIGs have the important responsibility of setting milestones for stability and completeness.
-Hopefully the content previously contained in [umbrella issues][] will be tracked in the `Graduation Criteria` section.
+Consider including folks that also work outside the sig or subproject.
+
+## Design Details
+
+### Test Plan
+
+**Note:** *Section not required until targeted at a release.*
+
+What is the test plan for this set of functionality e.g., e2e, integration, unit testing?
```

Comment:
Do you need a test plan for unit tests?

Reply:
I think the test plan should require some semblance of unit testing, but not sure that the inverse should be required. Admittedly, this is squishy, so I'll let @spiffxp of @timothysc chime in.

Reply:
In an ideal world, unit tests are always there. I guess I was bringing that expectation instead of requiring that each KEP state "there will be unit tests"... or were you thinking of something more detailed?

Reply:
I would be more passive-aggressive here: "Will there be e2e and integration tests in addition to unit tests?" :)